openassessment/mobile_assessment/views.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
```

Comment:
let's make sure we need this view

Reply:
removed, that was for API interface which we are not using now.